MSAL/src/native_auth/public/state_machine/state/ResetPasswordStates.swift

```diff
@@ -32,23 +32,23 @@ public class ResetPasswordBaseState: MSALNativeAuthBaseState {
     init(
         controller: MSALNativeAuthResetPasswordControlling,
         flowToken: String,
-        inputValidator: MSALNativeAuthInputValidating = MSALNativeAuthInputValidator()
+        inputValidator: MSALNativeAuthInputValidating = MSALNativeAuthInputValidator(),
+        correlationId: UUID
     ) {
         self.controller = controller
         self.inputValidator = inputValidator
-        super.init(flowToken: flowToken)
+        super.init(flowToken: flowToken, correlationId: correlationId)
     }
 }
 
 /// An object of this type is created when a user is required to supply a verification code to continue a reset password flow.
 @objcMembers public class ResetPasswordCodeRequiredState: ResetPasswordBaseState {
     /// Requests the server to resend the verification code to the user.
     /// - Parameters:
```

Comment:
just for consistency, can you please write the comment like this `/// - Parameter delegate: Delegate that receives callbacks for the operation` when a function has only one parameter?

Reply:
Done.